src/Faithlife.Data/DbConnectorCommand.cs

```diff
@@ -381,6 +387,9 @@ string Replacement(Match match)
 				command = CreateNewCommand();
 			}
 
+			if (CommandType != CommandType.Text)
+				command.CommandType = CommandType;
```

Comment:
I should have noticed this before, but this should be in `CreateNewCommand` below so we don't needlessly reset it for cached commands.

Reply:
Good catch! 👍🏼